templates/layout.html

```diff
@@ -39,16 +37,16 @@
 <main>{% block main %}{% endblock %}</main>
 
 <footer>
-  <nav class="wrapper">
-    <a href="mailto:contact+website@neomad.org">Contact us</a>
-    <a href="https://github.com/neomad-team/neomad.org">Github repository</a>
+  <nav class=wrapper>
+    <a href=mailto:contact+website@neomad.org>Contact us</a>
+    <a href=https://github.com/neomad-team/neomad.org>Github repository</a>
   </nav>
 </footer>
 
 {% with messages = get_flashed_messages(with_categories=true) %}
 {% if messages %}
 {% set message = messages[0] %}
-<script>alert("{{message[0]}}", '{{message[1]}}')</script>
+<script>alert('{{message[0]}}', '{{message[1]}}')</script>
```

Comment:
Using double quotes  on `message[0]` because it may contain `'`.
If you want to use single quotes maybe [you could try this](https://stackoverflow.com/questions/17941109/escaping-quotes-in-jinja2?answertab=votes#tab-top)